resources/js/components/Notifications/Notification.vue

```diff
@@ -31,12 +31,18 @@
         data: () => ({
             message: null,
             type: null,
-            show: null,
+            show: null
         }),
         methods: {
             close() {
                 this.show = false
+            },
+            format(str, arr) {
```

Comment:
I don't know if this function should be elsewhere because i don't know if any other api feedback gives these types of messages.